### DISCUSSION TITLE
Seeking Clarification on Cypress Tool Support Status

### DISCUSSION THREAD

**Discussion by mkathu** [cypress-io/cypress#28190] [General]
I am using cypress to do a poc for one of the projects. I am really happy with the features or capabilities that cypress provides.

I've heard rumors about the discontinuation of Cypress support or restrictions on its usage. Can you please confirm the current status, reasons for any changes, and provide a timeline or roadmap for support? We're considering it for our project and want to make informed decisions. Thank you.

**Comment by jennifer-shehane** [comment]
@mkathu Cypress is still under full support. We release every 2 weeks. You can view our current [changelog](https://docs.cypress.io/guides/references/changelog) to see updates. The founder of our company gave a talk a couple of weeks ago at Cypress Conference outlining our roadmap - you can watch it [here](https://www.youtube.com/watch?v=Br7mjzHIII0&list=PL8GlT7H3xOcI_CeBSml3tRhKx4kgVYLJa&index=2). You can see a portion of the Cypress App roadmap [here](https://github.com/orgs/cypress-io/projects/13) at any time.
